hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedSSTDumpIterator.java

```diff
@@ -244,40 +223,45 @@ protected void finalize() throws Throwable {
    * Class containing Parsed KeyValue Record from Sst Dumptool output.
    */
   public static final class KeyValue {
-    private String key;
-    private Integer sequence;
+    private byte[] key;
+    private BigInteger sequence;
```

Comment:
Is `Long` sufficient here? `BigInteger` iirc is a lot slower than `Long` or `Integer`. It is concerning when it is used as a counter in the iterator.

Reply:
done